core/src/main/java/hudson/security/SecurityRealm.java

```diff
@@ -580,6 +573,21 @@ public Filter createFilter(FilterConfig filterConfig) {
         private Object readResolve() {
             return NO_AUTHENTICATION;
         }
+        
+        @Extension
```

Comment:
Similar to the "No SCM" option in jobs this should set ordinal to always appear last in the list.